test/unit/fake/initialization.spec.ts

```diff
@@ -90,4 +92,21 @@ describe('Fake: Initialization', () => {
     const fake = await smock.fake(storageArtifact);
     expect(fake.store._watchable).not.to.be.undefined;
   });
+
+  it('should throw error for invalid json string abi', async () => {
+    await expect(smock.fake(`{invalid}`)).to.be.rejectedWith(
+      Error,
+      `unable to generate smock spec from abi string, Unexpected token i in JSON at position 1`
+    );
+  });
+
+  it('should throw error for non existent contract', async () => {
+    let regex: RegExp = /unable to generate smock spec from contract name*/;
```

Comment:
Regex only covers the portion of the string that is controlled by the smock package because 3rd party library error codes / error messages are subject to change at any point and an exact string match on that value is not recommended.

Reply:
Could we include the contract name in the error?